apps/music/js/ui/views/tiles_view.js

```diff
@@ -124,7 +127,8 @@ var TilesView = {
     var INITIALLY_HIDDEN_TILE_WAIT_TIME_MS = 1000;
 
     var setTileBackgroundClosure = function(url) {
-      tile.style.backgroundImage = 'url(' + url + ')';
+      container.appendChild(albumArt);
```

Comment:
I think it would make more sense to append the `<img>` element immediately, rather than waiting for this function to be called.